src/dsp_tools/commands/excel2json/models/input_error.py

```diff
@@ -284,3 +284,29 @@ def execute_error_protocol(self) -> str:
         ]
         msg.extend([problem.execute_error_protocol() for problem in self.problems])
         return separator.join(msg)
+
+
+@dataclass(frozen=True)
+class ListSheetComplianceProblem:
+    sheet_name: str
+    problems: dict[str, str]
+
+    def execute_error_protocol(self) -> str:
+        msg = [
+            f"The excel sheet '{self.sheet_name}' has the following problem(s):",
+        ]
+        msg.extend([f"Field: '{key}', Problem: {value}" for key, value in self.problems.items()])
```

Comment:
What is "Field" in this context? In the code, you provide a short description as the key for the dict, and not a "field". So I wonder if the final error message will be understandable.

Reply:
Follow up question: Have you checked that the final error message looks like you intended?

Reply:
That's fine, but I still wonder if "field" is the right term here.